near-contract-standards/src/fungible_token/events.rs

```diff
@@ -0,0 +1,215 @@
+pub struct FtBurn<'a> {
+    pub owner_id: &'a AccountId,
+    pub amount: &'a U128,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub memo: Option<&'a str>,
+}
+
+impl FtBurn<'_> {
+    /// Logs the event to the host. This is required to ensure that the event is triggered
+    /// and to consume the event.
+    pub fn emit(self) {
+        Self::emit_many(&[self])
+    }
+
+    /// Emits an FT burn event, through [`env::log_str`](near_sdk::env::log_str),
+    /// where each [`FtBurn`] represents the data of each burn.
+    pub fn emit_many<'a>(data: &'a [FtBurn<'a>]) {
+        new_141_v1(Nep141EventKind::FtBurn(data)).emit()
+    }
+}
+
+#[derive(Serialize, Debug)]
+pub(crate) struct Nep141Event<'a> {
+    version: &'static str,
+    #[serde(flatten)]
+    event_kind: Nep141EventKind<'a>,
+}
+
+#[derive(Serialize, Debug)]
+#[serde(tag = "event", content = "data")]
+#[serde(rename_all = "snake_case")]
+#[allow(clippy::enum_variant_names)]
+enum Nep141EventKind<'a> {
+    FtMint(&'a [FtMint<'a>]),
+    FtTransfer(&'a [FtTransfer<'a>]),
+    FtBurn(&'a [FtBurn<'a>]),
+}
+
+fn new_141<'a>(version: &'static str, event_kind: Nep141EventKind<'a>) -> NearEvent<'a> {
+    NearEvent::Nep141(Nep141Event { version, event_kind })
+}
+
+fn new_141_v1(event_kind: Nep141EventKind) -> NearEvent {
+    new_141("1.0.0", event_kind)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use near_sdk::{test_utils, AccountId};
+
+    fn bob() -> AccountId {
+        AccountId::new_unchecked("bob".to_string())
+    }
+
+    fn alice() -> AccountId {
+        AccountId::new_unchecked("alice".to_string())
+    }
+
+    #[test]
+    fn ft_mint() {
+        let owner_id = &bob();
+        let amount = &U128(100);
+        FtMint { owner_id, amount, memo: None }.emit();
+        assert_eq!(
+            test_utils::get_logs()[0],
+            r#"EVENT_JSON:{"standard":"nep141","version":"1.0.0","event":"ft_mint","data":[{"owner_id":"bob","amount":"100"}]}"#
+        );
+    }
+
+    #[test]
+    fn ft_mints() {
+        let owner_id = &bob();
+        let amount = &U128(100);
+        let mint_log = FtMint { owner_id, amount, memo: None };
+        FtMint::emit_many(&[
+            mint_log,
+            FtMint { owner_id: &alice(), amount: &U128(200), memo: Some("has memo") },
+        ]);
+        assert_eq!(
+            test_utils::get_logs()[0],
+            r#"EVENT_JSON:{"standard":"nep141","version":"1.0.0","event":"ft_mint","data":[{"owner_id":"bob","amount":"100"},{"owner_id":"alice","amount":"200","memo":"has memo"}]}"#
+        );
+    }
+
+    #[test]
+    fn ft_burn() {
+        let owner_id = &bob();
+        let amount = &U128(100);
+        FtBurn { owner_id, amount, memo: None }.emit();
+        assert_eq!(
+            test_utils::get_logs()[0],
+            r#"EVENT_JSON:{"standard":"nep141","version":"1.0.0","event":"ft_burn","data":[{"owner_id":"bob","amount":"100"}]}"#
+        );
+    }
+
+    #[test]
+    fn nft_burns() {
```

Comment:
`ft_burns`

Reply:
Oh it's merged 😅 I just noticed that
Created PR with the fix https://github.com/near/near-sdk-rs/pull/735